app/assets/javascripts/settings-controller.coffee

```diff
@@ -59,14 +60,40 @@ template = """
       <option value={{code}}>{{description}}</option>
       {{/}}
     </select>
-  </div>
-
+  </div>                           
 </div>
+
+
+<h1 class="settings-subheader">Works in Progress</h1>
+<a><center> Here you can access all the models that have been edited</center></a>
+  
+<ul>
+  {{#each workInProgressLinks}}
+    <li> <a href="{{url}}">{{modelTitle}}</a> <br>Date Last Modified: {{dataAccessed}}</li>
+    <br>
+  {{/each}}
+</ul>
+
 """
+formatLinks = (wipStorage) ->
+  result = []
+
+  for url, model of wipStorage.inProgress
+    if model.title and model.timeStamp
+      formattedUrl = "/launch#http://" + url.replace(/^url:\/\//, '')
```

Comment:
This still assumes the `http://` protocol, but if we're viewing a secure page it needs to be `https://` instead.